flink-table/flink-sql-client/pom.xml

```diff
@@ -152,7 +161,9 @@ under the License.
 						<configuration>
 							<artifactSet>
 								<includes combine.children="append">
-									<include>org.apache.flink:flink-table_${scala.binary.version}</include>
+									<include>org.apache.flink:flink-table-common</include>
+									<include>org.apache.flink:flink-table-api-java-bridge_${scala.binary.version}</include>
```

Comment:
why is the scala bridge not included?

Reply:
Because the SQL Client uses the Java Table API.